### PR TITLE
Better elevators

### PR DESCRIPTION
**Pull request by ShaAlexander**
**Describe Pull request**
Make them all work on both sides, the most right elevator takes you to the roof, the others takes you to the basement on both sides.

If your PR is to fix an issue mention that issue here

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? Yes, i use for a long time.
- Does your code fit the style guidelines? Yes, it was a matter of copy and paste, then edit ;-) 
- Does your PR fit the contribution guidelines? Yes.

### CODE REVIEW

File: client/job.lua [hunk 1]
```diff
@@ -427,11 +427,59 @@ CreateThread(function()
                 end
             end
 
+            for k, v in pairs(Config.Locations["main2"]) do
+                local dist = #(pos - v)
+                if dist < 1.5 then
+                    sleep = 7
```
Comment (PsychoShock) [inline client/job.lua:433]: Why sleep is `7`, should be at `0`

File: client/job.lua [hunk 2]
```diff
@@ -401,13 +401,13 @@ CreateThread(function()
...
-            local currentHospital = 1
+           local currentHospital = 1
 
             for k, v in pairs(Config.Locations["main"]) do
                 local dist = #(pos - v)
                 if dist < 1.5 then
-                    sleep = 5
-                    DrawText3D(v.x, v.y, v.z, Lang:t('text.elevator_roof'))
+                    sleep = 7
+                    DrawText3D(v.x, v.y, v.z, "~g~E~w~ - Take the elevator to the roof")
```
Comment (PsychoShock) [inline client/job.lua:410]: Wrong, you speak english but some people using this script don't.

**Review by PsychoShock — CHANGES_REQUESTED** [review]
Logic is wrong on this, it should be re-worked correctly. Also please change all sleep's to `0` when you have a 3d text. Please use the translation system for everyone.

File: client/job.lua (hunk 2 above)
Comment (ShaAlexander) [reply client/job.lua:410]: since when is ML a thing?

File: client/job.lua (hunk 1 above)
Comment (ShaAlexander) [reply client/job.lua:433]: did not change anything here just copy paste then made some edits.

**Comment by ShaAlexander** [comment]
since when i

**Pull request closed without merging**